### PR TITLE
Module_2.5

### PR DESCRIPTION
**Pull request by andrew394**

### CODE REVIEW

File: taf/src/main/java/com/epam/gomel/tat/page/YandexDiskCreateFolderPage.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class YandexDiskCreateFolderPage extends AbstractPage {
+
+    private static final By FOLDER_NAME_BOX = By.xpath("//form[@class='rename-dialog__rename-form']//input");
+    private static final By FOLDER_SAVE = By.xpath("//button[contains(.,'Сохранить')]");
+    private static final By CREATED_FOLDER = By.xpath("//div[text()='Вы создали папку ']");
```
Comment (AndreiValnianka) [inline taf/src/main/java/com/epam/gomel/tat/page/YandexDiskCreateFolderPage.java:15]: What will happen if a user uses Belarus or English on his browser?

File: taf/src/main/java/com/epam/gomel/tat/page/YandexDiskFolderPage.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    public boolean isDocumentInsideAppropriateFolder() {
+        By documentInsideAppropriateFolderBy = By.xpath(String.format(CHOOSE_CREATED_DOCUMENT, name));
+        return isThePageOpened(documentInsideAppropriateFolderBy);
+    }
+
+    public YandexDiskTextDocumentReopenPage reopenTextDocument() {
+        By textDocumentBy = By.xpath(String.format(CHOOSE_CREATED_DOCUMENT, name));
+        Actions actions = new Actions(driver);
+        actions.doubleClick(waitForElementLocatedBy(textDocumentBy)).build().perform();
+        switchToTab(1);
```
Comment (AndreiValnianka) [inline taf/src/main/java/com/epam/gomel/tat/page/YandexDiskFolderPage.java]: Please avoid "magic numbers".  Maybe make sense to check the quantity of tabs before double-clicking and try to perform switching to the next?

File: taf/src/main/java/com/epam/gomel/tat/test/YandexDiskCreateTextDocumentTest.java [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.*;
+
+import static org.testng.Assert.assertTrue;
+
+public class YandexDiskCreateTextDocumentTest {
+
+    private static final String LOGIN = "andrew-394";
+    private static final String PASSWORD = "andrew394";
```
Comment (AndreiValnianka) [inline taf/src/main/java/com/epam/gomel/tat/test/YandexDiskCreateTextDocumentTest.java]: Do login/password public and move in an appropriate place (I suppose right now it is AbstractScreen)

File: taf/src/main/java/com/epam/gomel/tat/test/YandexDiskLoginTest.java [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    public void verifyLoginTest() {
+        boolean isLoginCorrect = passportYandexPage
+                .enterPassword(PASSWORD)
+                .enterTheMenuOfFilesPage()
+                .isSettingsEnabled();
+        assertTrue(isLoginCorrect);
+    }
+
+    @Test
+    public void verifyLoginIncorrectTest() {
```
Comment (AndreiValnianka) [inline taf/src/main/java/com/epam/gomel/tat/test/YandexDiskLoginTest.java]: verifyIncorrectLoginTest()

File: taf/src/main/java/com/epam/gomel/tat/test/YandexDiskMenuTest.java [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+        driver.quit();
+        driver = null;
+    }
+
+    @Test
+    public void verifyMenuOfFilesTest() {
+        boolean isMenuOfFilesCorrect = yandexDiskMenuOfFilesPage
+                .chooseTheMenuOfFiles()
+                .isMenuOfFilesEnabled();
+        assertTrue(isMenuOfFilesCorrect);
```
Comment (AndreiValnianka) [inline taf/src/main/java/com/epam/gomel/tat/test/YandexDiskMenuTest.java]: just 
assertTrue(yandexDiskMenuOfFilesPage
                .chooseTheMenuOfFiles()
                .isMenuOfFilesEnabled();)

**Review by AndreiValnianka — CHANGES_REQUESTED** [review]
see comments

File: taf/src/main/java/com/epam/gomel/tat/page/YandexDiskCreateFolderPage.java (hunk 1 above)
Comment (andrew394) [reply taf/src/main/java/com/epam/gomel/tat/page/YandexDiskCreateFolderPage.java:15]: I changed the browser language to English. The page of Yandex Disk still loads in Russian. Tests will be correct.

**Comment by andrew394** [comment]
I corrected your comments.

**Review by AndreiValnianka — APPROVED** [review]

**Pull request merged**